lib/ansible/plugins/filter/combine.yml

```diff
@@ -34,9 +34,16 @@ DOCUMENTATION:
 EXAMPLES: |
 
     # ab => {'a':1, 'b':3, 'c': 4}
-    ab: {{ {'a':1, 'b':2} | combine({'b':3, 'c':4}) }}
+    ab: {{ {'a':1, 'b':2} | ansible.builtin.combine({'b':3, 'c':4}) }}
 
-    many: "{{ dict1 | combine(dict2, dict3, dict4) }}"
+    many: "{{ dict1 | ansible.builtin.combine(dict2, dict3, dict4) }}"
+    
+    # Nested merge, I.E. Defaults using complex data structures with partial customization overwrites
+    # Note: Keyword parameters need to be listed *after* positional parameters
```

Comment:
This shouldn't be explained in each plugin. Also, this is already covered in documentation:

https://docs.ansible.com/ansible/latest/plugins/filter.html#using-filter-plugins

> When passing both types, positional arguments should go first.

```suggestion
```

Reply:
Can we tweak the page generator to add this note to every filter plugin that uses both types?
Maybe where currently the text starting with "This describes keyword parameters of the filter. " is generated?

Reply:
Moved this note to: https://github.com/ansible-community/antsibull-docs/pull/101